readinessProbe/main.go

```diff
@@ -0,0 +1,65 @@
+// Description: This program creates a web server to verify if the managedOCS
+//              resource is ready. It is used as a readiness probe by the
+//              ocs-osd-deployer operator. For this to be set up, the following
+//              sidecar should be added to the manager CSV:
+//      - name: readinessServer
+//        command:
+//        - /readinessServer
+//        image: quay.io/rcampos/ocs-osd-deployer-readiness:latest
+//        readinessProbe:
+//          httpGet:
+//            path: /readyz
+//            port: 8081
+//          initialDelaySeconds: 5
+//          periodSeconds: 10
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis"
+	v1 "github.com/openshift/ocs-osd-deployer/api/v1alpha1"
+	"github.com/openshift/ocs-osd-deployer/readinessProbe/readiness"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func main() {
+	// Setup client options
+	var options client.Options
+
+	// The readiness must have these schemes to deserialize the k8s objects
+	options.Scheme = runtime.NewScheme()
+	utilruntime.Must(clientgoscheme.AddToScheme(options.Scheme))
+	utilruntime.Must(ocsv1.AddToScheme(options.Scheme))
+	utilruntime.Must(v1.AddToScheme(options.Scheme))
+
+	k8sClient, err := client.New(config.GetConfigOrDie(), options)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
I think it will be beneficial for us to have  please  surrounding message on top of the error to make it easier to track in the logs